flocker/node/agents/blockdevice.py

```diff
@@ -295,6 +295,8 @@ class BlockDeviceVolume(PRecord):
     """
     blockdevice_id = field(type=unicode, mandatory=True)
     size = field(type=int, mandatory=True)
+    # Replace with cloud_instance_id
```

Comment:
Remember that this backend may be used by non-cloud things. Maybe `storage_backend_id`?